code/modules/mob/living/silicon/robot/examine.dm

```diff
@@ -3,6 +3,9 @@
 	if(desc)
 		. += "[desc]"
 
+	var/model_name = model ? "\improper [model.name]" : "\improper Default"
```

Comment:
why do you need the improper

Reply:
The improper is for the usage of the `\an` macro, otherwise it just displays nothing.